02Lesson/01Project/index.html

```diff
@@ -1,9 +1,33 @@
zTMXXl7Hh1hYk5XiLmL6XW9efw9t3DOaMjcRvb+E2rQ+Z9MAbHTkRG91nUO0A8GPG+unqpcFiw0icwYAEmNraa3OSvpLLEuJoD9BkiC0NJnT+4mbHLwWO3EukucM4mTGc2vnb3K3+0PqpyDwuzBjI887Z2WEKAaHBwBJDbi0m2Q0zItzzWeo3/Pv17afZNVzaNWZI+oDPM6wrPwrSL2tg2AvYayY8df2VXC1Q2lwMEkkkAnMQM7WuO9XfhCQ5zYMRcG0Fp5nn5piW+q3mtfSYA365dtBgnOZyAn15Ky56jrYhjQXOe1rRmSYufvdNc/LqjinYbJXFNBTC5ArioiUOco3OViEe5VsQbW3Cke5V6lRdIxTahylV6pjmn1HKniXHfr78lplFiapsGxO50y99yo4mpwiALRBuRNtBqYv5qxXfY3gm0jP3CzMVXEm5kk22BkAAaW9CrEZ+PJLiXCbSRzAIDZN9fLoub7YqCST+pw8BYrc7QcTYDI38rT4nlYLmO06su99EFIpEIUaIhCECpwTUqDocI+WNM6D8FW6by1pOl2z1g2Onr4rK7LfLY2Pr7KvMqESNCII0IsYPeAeoC83Uyvp/z68uJWnRxLiDfnHO0nYT+FNSxIP0kAg5/vp/KyBU3P75CFZpul3ESQBrE7whY0H1PlQWGAZnUXjyt5K9gKzi8v4cx9WV5ykHr0MrDr12kyPC8DxvM+9FLh65EjjOehtrMQYvyUl9p1zvLp6OKc55YXXIBDf7rXiM7jbqmnEM4i0yeZBvPWIOkzmFC3tqg+l9bTxgyBaC8wA8HOYF+pzyGM/tH5hLSDwzYD6JkEXknhED3muuvPObvx0HY73B5AbxG4AB1k6mM7XWzisY5lH51IDjggixvsdiI+y47EdqFplg4Ys08MGNSCDIMg3vmbrU7K7apNY9tZoDXaOgTO855Sn+ks96t0u3zVDWFxHE4XAAIJIjLMA7xK3uz6LqbGsJnh1/xHMkiLXJtfqvP6FM8XC0F0HQE2Fg47A812HZuOLRwVHHivE5nQ5bWzgqHcn43WusguULKgIsQRySFyORznKF7kOcoXPVQPeq73JKjlA6otxmnVXc/uqFZ5uQfL35KWq9UMRWgEnTwWmIhq1rkanK+fd7z8KGKrm4nbvMHOcsz4KV9W58PcrLxdbhnmWg3vEwRIysfTkqql2jiv1G4mwygaacpv8AgrmnFaPaFVv6WtDYyz1iYmfcLMQgSJUiihCEIFSpqVBd7LqQ+N7d+nvmtYrnmOgyFv0anE0OGo/lcf6T9ev/AM3frxSARc+Gv7JPnGIFgfYvmmufv48tk0Lm9ILlIKhy996jLwMgZ3Mf+v5JSPrF2d/DppyVRJVOV+fTl1Q1+VuhEKIZJGuyHhoiVoUcQ3J8gQQLBx34SSW2O4vl0T8PjS27RnbhI+Y0zyJF+nks8ROZ0nfmbnyVo4lwc4CHN2feRmHO4TBIGkkCYWtYsa/ZPxA6m4NLvoLhxDrnHiV11em6A6mGE6O/yzNjqfyvL6laZJH1EzIMD/bHgu0+D+1OKkab3XZccm/tdace5nuNipXeXANqhjbAN4c5v+q2gPjyK06dQxcyVzuIpOe4OY4EWgzOom0QLDfUqShVc1/1PkXtlfO88sv2Rzrde9QOqKsMcx0QbnIct0j6irJ1R6qPqJz6ipvetRmnVavO+n5VDEvHeNeiK2JzWfXxI3WmEdZ0ZmctQJ75A1WP2hiYm/pl+crKfF4vY38+46fvosDF1uIov1FVqFxk+QAHgExCEUJEIQCEIQCEIQKFo9lV7lh1uOu3f9lmpzSpZsxrnq83Y3yhoF57k/BPFVkj9QzHP8H1so3BcLMe7nuWbEZSJxCQKLpwdp7/AJTUZJAqam4IgzmPuR9kOdYkH9WbQCNZ6RZFZxhsjJsDmJN/UKEX1VZtI4K32bjflv4jBmxkTIP6mjaQVWcOc+NuSTgWo512/YVak0u+W9zg6/CQBw7mwtaPBaGMpA5AGTeYj3+VxWDw7w/iaDw63iREkFwhdFh8WXCBYiYGdpMEzktRw6TYR1jvM9ItHT8q26rbzVSOEWN+6/go3VICYzanqVlUqVVHVqqnXrLUZtUu0sQZN4CxMRiCDmRHj0V3F1b5iTr6DO5/CyMS+5j3uimVqxOqgQhVAhCRAIQhAqRCEAhCEAhCEFnA4s03Bw6EbjULpHBtRvzGZHyOoPOVySu9nY91J0i4Obd/3Us1rnu8th9FVyFpUsSyqJB/IPMKLFU220O+h/fquV5ejn+mqMJIVlzQCn4WiC6ItpHPIpjXmhqg92Yv5pgZbI9dlr4lwDRa1oNp0UHyDBDiY/HVXGPP0pmjFyDGlpzU1CkARxEwdhl42Wn8mQNNhYdIGyQ0zBEznzWscr2XBVHf22ZfPXSTGa0WuaBGVotzVZlX6ZJvqBeTE92frsoalXhlxA3AkeZWmLV2vXgZkAa9N1R/4m8ffwtprt0VPE1y9ogkAznxHa8W9wqz8RB4ZykkkiZnYZZj2ExGhWxCoYjE7qnXxZHJZ1fFE6qs/UuKxc78lQSkpEagQhCBEIQgEIQgEIQgEIQgEIQgEIQgkpVS0yDC1MP2oCIf/Kx0qDfpVmkkTbT91LQfwkHz7+Xeuda8jIqenjCPfrupi+Vdc+tIMGYMe90lCsC0cW53tqudb2nIg7R3d1x3JR2iLCdLm/Kwk7bpia6X50wcvXaVGcTAMddVz57SnXKN97qOr2iTrOfdtmERttxzW2cdz43gaqB9aW5ls3FrxvEysI4wzy8LdyYcQYIAAnOJy2zVGpiarj/idYXcQ3LZuaoVcWco8VWe8nMymoYUlIhCKEJEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBKkQgVCEIBCEIBCEIBCRCBUJEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBKhCASIQgEIQgVIhCAQhCAQhCAQhCAQhCAQhCAQhCAQhCD//Z"/>
```

Comment:
Better src attribute with link to image file, for better performance and cleaner HTML code. 
Alt attribute missing